### ISSUE TITLE
[Accessibility]: Opening then closing code intel popover changes DOM structure

### ISSUE THREAD

**Issue by varungandhi-src** [sourcegraph/sourcegraph#36565]
### Audit type

Screen reader navigation

### User journey audit issue

https://github.com/sourcegraph/sourcegraph/issues/33509

### Problem description

On performing a screen reader click on an occurrence in the blob view, the DOM structure changes to break a link for `"fmt"` (in a Go import) into three ranges with their own links (one for `"`, one of `fmt` and one for `"`).

Additionally, none of these clicks actually work (my guess is that they're marked clickable due to https://github.com/sourcegraph/sourcegraph/issues/35267).

This means that it isn't possible to trigger the popover on the same import again without a page reload.

(Putting this under "Go to Definition" but it also applies to "Find References" and related user journeys.)

### Expected behavior

The DOM structure should return to its earlier state, so that clicking again on the same import triggers the popover again.

### Assigning labels

- [ ] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [ ] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [ ] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by my team, I have assigned a relevant member to it, or I will do so in the near future.

**Comment by felixfbecker** [comment]
I think with the new keyboard support in the blob view this doesn't apply anymore.
